src/services/message.service.ts

```diff
@@ -1,15 +1,18 @@
-import { IMessage } from '../common/types';
+import { IMessage, IEmojiTable } from '../common/types';
 import { GuildChannel, Guild, TextChannel, MessageEmbed, MessageReaction, User } from 'discord.js';
 import { GuildService } from './guild.service';
 import Constants from '../common/constants';
 import { LoggerService } from './logger.service';
+import * as moment from 'moment';
 
 export class MessageService {
   private _botReportingChannel: TextChannel | null = null;
   private _guild: Guild;
   private _linkPrefix: string = 'https://discord.com/channels';
   private _ARROWS = ['⬅️', '➡️'];
 
+  private _TWO_MINUTES: number = moment.duration(2, 'minutes').asMilliseconds();
```

Comment:
Just put this inline where it goes

Reply:
it gave me errors when I did that for some reason I have no idea why.

Reply:
What error do you get when you do
`time: moment.duration(2, 'minutes').asMilliseconds(),`

Reply:
and it seems to be working now :sob: I literally copied the code up to their.